internal/sdks/sdks.go

```diff
@@ -22,8 +22,6 @@ func ReplaceFlagKey(instructions string, key string) string {
 		key,
 		"my-boolean-flag",
 		key,
-		"FLAG_KEY",
```

Comment:
There aren't any "FLAG_KEY" values to replace with the new SDK instruction examples.

Reply:
I think we can get rid of all of these except for the ones above the comment line now that we've made them all the same value?

Reply:
same thing with sdk key -- I forgot to clean that up when I removed the old instructions.